server/aws-lsp-codewhisperer/src/language-server/chat/telemetry/chatTelemetryController.ts

```diff
@@ -200,6 +224,27 @@ export class ChatTelemetryController {
         )
     }
 
+    #enqueueCodeDiffEntry(params: InsertToCursorPositionParams) {
+        const documentUri = params.textDocument?.uri
```

Comment:
Currently this is not part of the interface therefore this field will be empty. Is it ok to have it empty until we figure out how we want to pass this information

Reply:
We are able to pass these through via telemetry. Maybe because it doesn't have defined types on the protocol side?